src/plans/create/steps/storage-map/TargetStorageField.tsx

```diff
@@ -36,7 +36,7 @@ const TargetStorageField: FC<TargetStorageFieldProps> = ({ fieldId, targetStorag
           >
             <SelectList>
               {targetStorages.map((targetStorage) => (
-                <SelectOption key={targetStorage.id} value={targetStorage.name}>
+                <SelectOption key={targetStorage.id} value={targetStorage}>
```

Comment:
its right to remove the name?

Reply:
Yes, this fixed an issue that @avivtur brought up to me in slack when he was running through the wizard, which was this:

https://github.com/user-attachments/assets/eeefd6dc-a40b-4e54-ab8f-90432a9e4986